src/LaravelFacebookAds/Console/App/GenerateTokenCommand.php

```diff
@@ -25,14 +25,23 @@ class GenerateTokenCommand extends Command
      * @var string
      */
     protected $description = 'Generate a new user token';
+    
+    
+       /**
+     * @param FacebookAdsService $facebookAdsService
+     */
+    public function handle(FacebookAdsService $facebookAdsService)
+    {
+        $this->fire($facebookAdsService);
+    }
 
     /**
      * Fire command
      *
      * @param FacebookAdsService $facebookAdsService
      * @return bool|void
      */
-    public function fire(FacebookAdsService $facebookAdsService)
+    public function fire($facebookAdsService)
```

Comment:
@LeorentKelmendi I'm not sure why you want to remove the typehint from the `fire` method?

Reply:
@nikolajlovenhardt  could be done both ways. Just did the fix since when I ran the command it sqwaks that no `handle` method was found.

Reply:
Yes indeed, but there's no reason to remove the type hint from the fire command.
I will merge it as soon as you re-add the type hint for the `fire` method